travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,28 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
```

Comment:
Well, as I mentioned a few times in scope of this pull request: if you expect to do something in scope of another task appropriate `FIXME` or `TODO` should be added.

In scope of definition of some arguments, they should be described when you expect to do so. If you don't expect to describe it in current review, they should be removed.